src/verinfast/agent.py

```diff
@@ -856,6 +860,11 @@ def main():
         print(f"""The log for this run has moved to:
               {d}/{new_folder_name}/{new_file_name}""")
 
+    # We only do this if you have a remote config but didn't upload
+    if agent.config.shouldUpload is False and agent.config.is_path_remote():
```

Comment:
is_path_remote() won't work. config.cfg_path gets overwritten from URL to local path

Reply:
I went ahead and fixed this.